hello-tribuo/src/main/java/com/gluonhq/samples/sample/Main.java

```diff
@@ -0,0 +1,142 @@
+package com.gluonhq.samples.sample;
+
+import com.gluonhq.attach.display.DisplayService;
+import com.gluonhq.attach.util.Platform;
+import com.gluonhq.charm.glisten.application.MobileApplication;
+import com.gluonhq.charm.glisten.control.AppBar;
+import com.gluonhq.charm.glisten.control.FloatingActionButton;
+import com.gluonhq.charm.glisten.mvc.View;
+import com.gluonhq.charm.glisten.visual.MaterialDesignIcon;
+import com.gluonhq.charm.glisten.visual.Swatch;
+import java.net.URL;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Dimension2D;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.chart.CategoryAxis;
+import javafx.scene.chart.Chart;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.StackedBarChart;
+import javafx.scene.chart.XYChart;
+import javafx.scene.chart.XYChart.Data;
+import javafx.scene.chart.XYChart.Series;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import org.tribuo.Model;
+import org.tribuo.MutableDataset;
+import org.tribuo.Prediction;
+import org.tribuo.classification.LabelFactory;
+import org.tribuo.classification.dtree.CARTClassificationTrainer;
+import org.tribuo.classification.evaluation.LabelEvaluation;
+import org.tribuo.classification.evaluation.LabelEvaluator;
+import org.tribuo.classification.sgd.linear.LogisticRegressionTrainer;
+import org.tribuo.common.tree.TreeModel;
+import org.tribuo.data.csv.CSVLoader;
+import org.tribuo.datasource.ListDataSource;
+import org.tribuo.evaluation.TrainTestSplitter;
+
+public class Main extends MobileApplication {
+
+    Series<String, Number> tpSeries = new Series<>();
+    Series<String, Number> fpSeries = new Series<>();
+
+    @Override
+    public void init() {
+        tpSeries.setName("TP");
+        fpSeries.setName("FP");
+        addViewFactory(HOME_VIEW, () -> {
+            FloatingActionButton fab = new FloatingActionButton(MaterialDesignIcon.SEARCH.text,
```

Comment:
If it is not used, better remove it. But since it is here, why don't we use it to trigger the `train` call? (It will help with nag screen, and chart animation)